app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -174,6 +176,18 @@ class MainViewModel @Inject constructor(
 
     private var currentThreadsLiveJob: Job? = null
 
+    fun checkWebViewVersion(showWebViewOutdated: Boolean) {
+        webViewOutdated.value = if (showWebViewOutdated) {
+            WebViewCompat.getCurrentWebViewPackage(appContext)?.versionName?.let { versionName ->
+                val majorVersion = versionName.substring(0, versionName.indexOfFirst { it == '.' }).toInt()
+                // Current major version of Android WebView System
```

Comment:
What does current mean in your comment?

Reply:
Current mean the latest version available on the store.